tracepoint_decode/src/enumerator.rs

```diff
@@ -217,43 +217,45 @@ impl<'inf> JsonMetaDisplay<'inf> {
         let tracepoint_name = self.event_info.tracepoint_name;
         let provider_name_end = if self
             .meta_options
-            .has(PerfMetaOptions::Provider.or(PerfMetaOptions::Options))
+            .has_flag(PerfMetaOptions::Provider.or(PerfMetaOptions::Options))
```

Comment:
Note: Nothing significant changed here. Just renamed `has` to `has_flag` for consistency.